src/components/Table.vue

```diff
@@ -22,26 +27,30 @@
 
 <script>
 import Pagination from '@/components/Features/Pagination.vue'
+import Search from '@/components/Features/Search'
 
 export default {
-  name: 'Vue_table_pro',
+  name: 'VueTablePro',
   props: {
     config: {
       type: Object,
       required: true
     }
   },
   components: {
+    Search,
     Pagination
   },
   data () {
     return {
       tableTitle: this.config.title || 'Features Title',
       tableData: this.config.data || [],
+      globalData: [],
       showData: [],
       tableHeaders: this.config.headers || [],
       pagination: this.config.pagination || null,
-      extraColumns: this.config.extraColumns || []
+      extraColumns: this.config.extraColumns || [],
+      search: this.config.search
```

Comment:
This property should have a fallback in case a user does not set search configuration